packages/calcite-components/src/components/action-menu/action-menu.e2e.ts

```diff
@@ -466,9 +466,43 @@ describe("calcite-action-menu", () => {
       expect(actions[2].getAttribute(activeAttr)).toBe(null);
 
       await page.keyboard.press("Enter");
+      await page.waitForChanges();
+
+      expect(await actionMenu.getProperty("open")).toBe(false);
+      expect(clickSpy).toHaveReceivedEventTimes(1);
+    });
+
+    it("should click the active action when clicked and close the menu", async () => {
+      const page = await newE2EPage({
+        html: html`<calcite-action-menu>
+          <calcite-action id="first" text="Add" icon="plus" text-enabled></calcite-action>
+          <calcite-action id="second" text="Add" icon="minus" text-enabled></calcite-action>
+          <calcite-action id="third" text="Add" icon="banana" text-enabled></calcite-action>
+        </calcite-action-menu> `,
+      });
 
       await page.waitForChanges();
 
+      const actionMenu = await page.find("calcite-action-menu");
+      const actions = await page.findAll("calcite-action");
+
+      expect(await actionMenu.getProperty("open")).toBe(false);
+
+      await actionMenu.callMethod("setFocus");
+      await page.waitForChanges();
+
+      await page.keyboard.press("ArrowDown");
+      await page.waitForChanges();
+
+      const clickSpy = await actions[0].spyOnEvent("click");
+
+      expect(await actionMenu.getProperty("open")).toBe(true);
+      expect(actions[0].getAttribute(activeAttr)).toBe("");
+      expect(actions[1].getAttribute(activeAttr)).toBe(null);
+      expect(actions[2].getAttribute(activeAttr)).toBe(null);
+
+      await page.$eval("calcite-action", (el: HTMLCalciteActionElement) => el.click());
```

Comment:
Is there a reason why this is calling `click` directly vs clicking via the E2E APIs? If so, can you add an explainer comment, otherwise, I'd suggest `await page.click("calcite-action");` to simplify.

Reply:
Yeah it wasn't triggering the closing of the menu when `await page.click("calcite-action")` was used.